packages/core/__tests__/auth/__system__/Logout.system.test.ts

```diff
@@ -17,6 +17,9 @@ import { Login } from "../../../src/auth/Login";
 import { Logout } from "../../../src/auth/Logout";
 import { ZosmfRestClient } from "../../../src/rest/ZosmfRestClient";
 
+// import non-exported modules
+const AuthOrder = jest.requireActual("../../../../imperative/lib/rest/src/session/AuthOrder").AuthOrder;
```

Comment:
This question is just for my own understanding...
Do we need to explicitly import `AuthOrder` this way? 

I do see it being exported as part of the `packages/imperative/src/rest/index.ts`